src/icons/fileIcons.ts

```diff
@@ -252,7 +252,8 @@ export const fileIcons: FileIcons = {
                 'cmd',
                 'awk',
                 'fish'
-            ]
+            ],
+            fileNames: ['activate', 'deactivate'],
```

Comment:
How are these file names related to the `console` file icon? I'd recommend removing this, it's too specific in my opinion. You can still add this in your user settings.